docs/release.md

```diff
@@ -0,0 +1,99 @@
+# Releases
+
+## Versionning
+
+### Example #1
+
+```
+| Polkadot   | v  0. 9.22    |
+| Client     | v  0. 9.22 0  |
+| Runtime    | v     9 22 0  |  =>  9220
+| senmver    |   00.09.22 0  |
+```
+
+### Example #2
+
+```
+| Polkadot   | v  0.10.42    |
+| Client     | v  0.10.42 0  |
+| Runtime    | v    10.42 0  |  => 10420
+| senmver    |   00.10.42 0  |
+```
+
+### Example #3
+
+```
+| Polkadot   | v  1.02.18    |
+| Client     | v  1.02.18 0  |
+| Runtime    | v  1 02 18 0  |  => 102180
+| senmver    |   01.02.18 0  |
+```
```

Comment:
`02`. That seems an odd version number in a semverish world...

Reply:
I just added the leading `0` to make it clear but indeed, this `0` usually never shows up.

Reply:
I removed all leading `0`.